src/main/java/com/datamountaineer/kcql/Kcql.java

```diff
@@ -516,7 +586,25 @@ public void exitColumn_name(ConnectorParser.Column_nameContext ctx) {
         if (isWithinIgnore[0]) {
           kcql.ignoredFields.add(field);
         } else {
-          kcql.addField(field);
+          List<String> cleanedParent = null;
```

Comment:
I think `cleanedParent` might be unnecessary? Can we not just use `nestedFieldsBuffer` directly?

Reply:
updated to set the parent as null if root element